### DISCUSSION TITLE
Saved states are always corrupt

### DISCUSSION THREAD

**Discussion by dbertoldi** [joncampbell123/dosbox-x#5269] [Q&A]
Hello. I don't know why but every game save state I capture is corrupt. I've tried both Prince of Persia 2 and Flashback and the majority loads but there are control problems, graphical corruptions and more, rendering it unplayable. If I load state without running the game itself I receive the message "program name mismatch". I'd be grateful for some help/advice. I'm running the latest 2024.10.01

Thanks.

**Comment by joncampbell123** [comment]
DOSBox-X only save states a subset of emulator state.
